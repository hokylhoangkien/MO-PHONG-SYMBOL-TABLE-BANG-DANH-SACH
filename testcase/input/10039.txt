INSERT o string
INSERT m string
INSERT n string
BEGIN
INSERT m string
BEGIN
PRINT
INSERT o string
END
END