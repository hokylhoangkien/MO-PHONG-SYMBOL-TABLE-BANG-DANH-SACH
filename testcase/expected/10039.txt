success
success
success
success
o//0 n//0 m//1
success